City-of-Helsinki/hel-fi-drupal-grants
# fix: AU-1705: Bank account confirmation file copying.

# [AU-1705](https://helsinkisolutionoffice.atlassian.net/browse/AU-1705)

## What was done
This pull request fixes an issue where bank account confirmation files would not get copied over to the new application when an application was copied.

## How to install
Make sure your instance is up and running on the correct branch.
  * `git checkout feature/AU-1705-bankAccountAttachmentCopying`
  * `make fresh`
  * `make drush-cr`

## How to test
- Login and select any user role.
- Fill out any application and submit it.
- View the submitted application and check that it has a bank account confirmation file.
- Copy the application.
- Either submit the copied application or save it as a draft. The application should have a bank account confirmation file.
- Check that the attachment is also present in ATV.
- Review the code.

[AU-1705]: https://helsinkisolutionoffice.atlassian.net/browse/AU-1705?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
